### PR TITLE
feat(/lib/theme): add Tailwind CSS colors to  `<Alert>`s

### PR DESCRIPTION
**Pull request by lirbre**
## Description

_(It's my first PR here lol. Looking forward to fix things if needed. :))_ 

This change starts adding all colors to components in Flowbite. I'll be partially updating the components until I got every of then with all colors from FlowbiteColors avaliable. 

Fixes # [473](https://github.com/themesberg/flowbite-react/issues/473) _(partially)_

## Type of change
- [X] New feature

## How Has This Been Tested?
Since it was a visual change I used the Storybook to validate if colors are changing as expected.

- [x] Storybook

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/lib/components/Alert/Alert.tsx
Comment (rluders) [inline src/lib/components/Alert/Alert.tsx:30]: You don't need to use `Pick<>` here if you are getting all the colors from the `FlowbiteColors` interface, you just need to `AlertColors extends FlowbiteColors`. You could even, or probably remove the `AlertColors` interface, and use the `FlowbiteColors` in its place.

**Review by rluders — CHANGES_REQUESTED** [review]

File: src/lib/components/Alert/Alert.tsx
Comment (lirbre) [reply src/lib/components/Alert/Alert.tsx:30]: Surely!

Comment (lirbre) [reply src/lib/components/Alert/Alert.tsx:30]: The problem here is that I didn't added "green", "yellow" and "red" because it would be redundant to "success", "error" and "warning".

I'll add them and implement that or just replace it with an Exclude.

Comment (lirbre) [reply src/lib/components/Alert/Alert.tsx]: Did the "redundant" way so I could implement FlowbiteColors directly. Also added the Storybook argTypes to make it easier to test.

File: src/lib/components/Alert/Alert.tsx
Comment (tulup-conner) [inline src/lib/components/Alert/Alert.tsx:42]: Can you revert the change to this line? We're good to go with this PR at that point

**Review by tulup-conner — CHANGES_REQUESTED** [review]

File: src/lib/components/Alert/Alert.tsx
Comment (lirbre) [reply src/lib/components/Alert/Alert.tsx]: done!

**Review by tulup-conner — APPROVED** [review]
LGTM

**Comment by tulup-conner** [comment]
I can't merge this without a re-review from you @rluders

**Review by tulup-conner — APPROVED** [review]

**Review by rluders — APPROVED** [review]
lgtm. Thank you.

**Pull request merged**
